CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 ## CHANGELOG
 
-### 4.0.0  (2022-03-01)
+### 4.1.0 (TBD)
```

Comment:
It might not be the case anymore but I used to look for this string in one of the scripts to determine the version to deliver. In doubt let's keep this format.

```suggestion
### 4.1.0 (release date TBD)
```